mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/enricher/ContentParameterDeclarationEnricher.java

```diff
@@ -85,7 +85,6 @@ private void configureDsl(ParameterDeclaration p) {
 
   private void defaultToPayload(ParameterDeclaration p) {
     if (p.getDefaultValue() == null) {
-      p.setRequired(false);
```

Comment:
why do you remove this line? you cannot set a default value without being sure that it actually is optional. Do not speculate with someone else setting this later or before. The method shouuld be consistent on its own